environment.yml

```diff
@@ -2,21 +2,31 @@ name: geocat-applications
 channels:
   - conda-forge
 dependencies:
+  # Content
   - cartopy
   - geocat-datafiles
   - matplotlib
   - netcdf4
   - numpy
-  - pre-commit
+  - pandas
   - python
   - xarray
+
+  # Jupyter
```

Comment:
Should `Jupyter` include `jupyterlab`? I use the `jupyter lab` command to run jupyter locally for dev

Reply:
I think we should limit the official environment to explicitly required packages.

I know some packages have optional dev requirements, I think that would be a better way of including packages like `jupyter lab`, but I don't think setting that up for this project would be worth it yet personally.

Locally I will (try to) keep two copies of environments like this, one with just the packages in the official environment that I use for testing and documentation generation, and another with some dev packages or packages I'm testing out in it.

Reply:
Sounds good 👍